graphics/cursorman.cpp

```diff
@@ -124,11 +122,9 @@ void CursorManager::popAllCursors() {
 
 void CursorManager::replaceCursor(const void *buf, uint w, uint h, int hotspotX, int hotspotY, uint32 keycolor, bool dontScale, const Graphics::PixelFormat *format, const byte *mask) {
 	PixelFormat pixelFormat;
-#ifdef USE_RGB_COLOR
```

Comment:
I do not understand this removal. If `USE_RGB_COLOR` is not defined, we force everything to CLUT8, but this and the following changes break it?

Reply:
That code doesn't actually convert the graphics data, so it would incorrectly pass a true colour image to the backend while claiming it's CLUT8.